### ISSUE TITLE
mocha-sidebar is not working.

### ISSUE THREAD

**Issue by FrostyLeaf** [maty21/mocha-sidebar#101]
### Description
Just, not working, no sidebar, no icon, script can be found but not working yet.

#### Steps to Reproduce
Install mocha-sidebar from "extensions"
Reload vsc
Open an mocha test file

#### Expected Results
A mocha sidebar or an icon of mocha-sidebar could be found. Or at least, the scripts could work.

#### Actual Results
No sidebar, no icon, and scripts could not work.

#### Settings file

I have done nothing with this yet.

### Versions

- VScode: 1.22.2
- Mocha SideBar: v0.19.0
- Node Version: v8.9.4

### OS version
- [ ] Linux 
- [x] Mac High Sierra 10.13.4
- [ ] Windows

<img width="1280" alt="default" src="https://user-images.githubusercontent.com/8349568/39907179-8dcbf59e-5519-11e8-853a-ce135383d3ba.png">

**Comment by yehiyam** [comment]
Could you set mocha.logVerbous in the settings and let us know what you see in the output pane

**Comment by FrostyLeaf** [comment]
OK, I'll do that.

**Comment by FrostyLeaf** [comment]
**Log (Shared)**:

```bash
[2018-05-11 12:59:35.047] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:35.154] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 12:59:38.134] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:38.140] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:38.232] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 12:59:41.476] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:44.184] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:44.192] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:44.201] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 12:59:44.335] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 12:59:44.343] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:00:15.006] [sharedprocess] [info] Starting to clean up unused language packs.
[2018-05-11 13:01:15.442] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:01:15.442] [sharedprocess] [info] Uninstalling extension: maty.vscode-mocha-sidebar-0.19.0
[2018-05-11 13:01:15.442] [sharedprocess] [info] Requesting for confirmation to uninstall extension maty.vscode-mocha-sidebar-0.19.0
[2018-05-11 13:01:15.443] [sharedprocess] [info] Routing dialog request to the client window:1
[2018-05-11 13:01:16.775] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:01:16.792] [sharedprocess] [info] Successfully uninstalled extension: maty.vscode-mocha-sidebar-0.19.0
[2018-05-11 13:01:17.962] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:17.991] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:01:20.674] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:20.698] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:20.718] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:01:22.565] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:22.600] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:22.652] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:01:22.658] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:01:22.661] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:23.927] [sharedprocess] [info] Scanned user extensions: 5
[2018-05-11 13:01:41.207] [sharedprocess] [info] Installing extension: vscode-mocha-sidebar
[2018-05-11 13:01:42.471] [sharedprocess] [info] Downloaded extension: vscode-mocha-sidebar
[2018-05-11 13:01:42.508] [sharedprocess] [info] Removed the extension from uninstalled list: maty.vscode-mocha-sidebar-0.19.0
[2018-05-11 13:01:42.517] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:01:42.518] [sharedprocess] [info] Extensions installed successfully: maty.vscode-mocha-sidebar
[2018-05-11 13:01:42.659] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:01:42.701] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:02:51.275] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:02:51.290] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:02:51.317] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:02:51.335] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:02:51.342] [sharedprocess] [info] Scanned user extensions: 6
[2018-05-11 13:02:51.460] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:02:51.463] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:02:51.466] [sharedprocess] [info] Scanned system extensions: 69
[2018-05-11 13:02:54.886] [sharedprocess] [info] Scanned user extensions: 6
```

**Log (Window)**:

```bash
[2018-05-11 13:01:15.431] [renderer1] [info] Requested uninstalling the extension maty.vscode-mocha-sidebar from window 1
[2018-05-11 13:01:15.443] [renderer1] [info] Uninstalling the extension maty.vscode-mocha-sidebar-0.19.0 from window 1
```

Seems unhelpful.

**Comment by FrostyLeaf** [comment]
Oh, here is my **User Settings**:

```json
{
    "http.proxy": "http://127.0.0.1:1087",
    "eslint.packageManager": "yarn",
    "npm.packageManager": "yarn",
    "mocha.logVerbose": true
}
```

**Comment by FrostyLeaf** [comment]
OK, the sidebar is appeared now, works fine. But the scripts seems still to be broken. Basically, it can't found the test files.

**Comment by maty21** [comment]
Can you please print the output from mocha extension it should appear on
the output tab

**Comment by FrostyLeaf** [comment]
```
____________________________________________________________________________
trying to searching for tests using these settings: 

    mocha path: mocha
    test files location: test/**/*.js
    files to ignore: **/.git/**/*,**/node_modules/**/*
    environmets: {}
    requires: []
    options:  {}


if you find anything wrong please change those default settings
____________________________________________________________________________
data send to parent. Exiting.
```

**Comment by maty21** [comment]
What is your test folder

**Comment by FrostyLeaf** [comment]
```zsh
➜  gnat-grpc-mock-server pwd
/Users/juno/Workspaces/gnat-grpc-cdc/gnat-grpc-mock-server
➜  gnat-grpc-mock-server ls test
interactions.json register-test.js
➜  gnat-grpc-mock-server
```

**Comment by FrostyLeaf** [comment]
And here is my **Workspace Settings**:

```json
{
    "mocha.files.glob": "test/**/*.js"
}
```

**Comment by FrostyLeaf** [comment]
<img width="222" alt="default" src="https://user-images.githubusercontent.com/8349568/39908637-98ad19ea-5521-11e8-86cc-3a1c870b4d64.png">
The sidebar works.

**Comment by maty21** [comment]
Nice, you can switch back to the new layout if you want .
if you like the extension you are more than welcome to 🌟 us and rate us :)

**Comment by FrostyLeaf** [comment]
Emmmm, I think u don't get it. I mean, the sidebar works, but the scripts don't.

**Comment by maty21** [comment]
What is the script

**Comment by FrostyLeaf** [comment]
<img width="633" alt="default" src="https://user-images.githubusercontent.com/8349568/39909068-98f0f0aa-5523-11e8-90ef-df5f2dfb1023.png">
Well, you mentioned that in [README](https://github.com/maty21/mocha-sidebar#usage-via-command-palette)

**Comment by maty21** [comment]
Oh ok you can open an issue we didn't give it attention since you can run
it via verious way code lens or side bar

**Comment by FrostyLeaf** [comment]
OK, thx a lot.

**Comment by maty21** [comment]
And please change issue title

**Comment by FrostyLeaf** [comment]
Get it.

**Comment by maty21** [comment]
closed issue #102  is opened
